.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
+        "source.fixAll.eslint": true
+    },
+	"eslint.validate": [
+		"javascript",
+		"typescript",
+		"svelte"
+	],
+    "files.associations": {
+        "*.css": "tailwindcss"
+    }
+}
```

Comment:
Please add a new line at the EOFs everywhere where this red thingy is there on Github, I hate to see this 😨

Reply:
I may have missed some, if I did please notify me which ones need more EOF-ing.